README.md

```diff
@@ -7,29 +7,34 @@ Node.js tools for Visual Studio 2017 is developed and managed here.
 
 
 Check out our documentation on [the NTVS wiki](https://github.com/Microsoft/nodejstools/wiki) 
-and our (old) [feature overview](https://channel9.msdn.com/events/Visual-Studio/Connect-event-2015/801) video on Channel 9.
+<!--and our (old) [feature overview](https://channel9.msdn.com/events/Visual-Studio/Connect-event-2015/801) video on Channel 9.
```

Comment:
I would remove any comments if there's no purpose for them.

Reply:
removed!